layouts/partials/community_links.html

```diff
@@ -0,0 +1,28 @@
+{{ $links := .Site.Params.links }}
+
+<section class="row td-box td-box--4 td-box--gradient td-box--height-auto linkbox">
+<div class="col-xs-12 col-sm-6 col-md-6 col-lg-6">
+<h2>Learn and Ask</h2>		
+<p>Learn more about FSH here:
+{{ with index $links "user"}}
+{{ template "community-links-list"  . }}
+{{ end }}
+</div>
+<div class="col-xs-12 col-sm-6 col-md-6 col-lg-6">
+<h2>Develop and Contribute</h2>
+<p>If you want to get more involved by contributing to FSH, join us here:
```

Comment:
The links point to GH for SUSHI, not FSH -- so it seems a little inaccurate to call it "contributing to FSH".  Also, I think we should have links for contributing to this site (if people want to enhance docs, provide more tutorials, etc), the playground, and eventually GoFSH.  But at least SUSHI and  _this site_ for now.